### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by piero-vic**
# Desktop Version
- Adapt to desktop version.
- Add navigation links.
- Refactor code.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -192,13 +236,11 @@ li {
   font-size: 15px;
   padding: 10px;
   background: rgba(255, 255, 255, 0.24);
-  margin: 12px 12px 0 0;
 }
 
 .card {
```
Comment (Alexoid1) [inline style.css:241]: - [ ] Kindly make the description and dark opacity in the background disappears on `hover` the element.
- [ ] The button `See project` just appears when the user hovers over the element.
Captura de pantalla_2021-09-23_17-33-46

File: index.html [hunk 2]
```diff
@@ -14,54 +14,71 @@
...
+          <li>
+            <a href="#"><img src="assets/mail.svg" alt=""></a>
+          </li>
+        </ul>
       </nav>
     </header>
 
     <!-- Hero Section -->
-    <section id="hero-section" class="dark-section d-flex">
+    <section class="padding-section d-flex dark-section hero-section">
```
Comment (Alexoid1) [inline index.html:36]: Captura de pantalla_2021-09-23_17-38-17
- [ OPTIONAL] This is how your project looks with the zoom value at 25% you can easily fix it by adding a max-width property to the project container and then align it to the center.

**Review by Alexoid1 — CHANGES_REQUESTED** [review]
Hi @piero-vic,

Impressive project!
You are doing a great job so far, and the project is looking good.

## Required Changes :recycle:
- For the project to get approved implement the changes mentioned below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️ 

Hi @piero-vic,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**